src/main/java/roomescape/exception/NotFoundReservationException.java

```diff
@@ -0,0 +1,10 @@
+package roomescape.exception;
+
+public class NotFoundReservationException extends RuntimeException{
```

Comment:
```suggestion
public class NotFoundReservationException extends RuntimeException {
```

자동 정렬 한번씩 돌리면 좋을 것 같아요~

Reply:
반영했습니다~